docs/tutorial/reading_writing.md

```diff
@@ -0,0 +1,336 @@
+# Reading and writing
+
+In this section, we will guide you through the process of opening and extracting information from files using our simple, intuitive API. Whether you are working with text, binary, or other formats, you will find the tools you need to seamlessly integrate file reading into your Scala project. Let us dive in and unlock the power of data stored in files!
+
+**Important:** To avoid tutorial hell, we recommend doing a really small project while you read this tutorial to place acquired knowledge into practice. It does not have to be fancy, but it should tap into your curiosity and challenge you just a little.
```

Comment:
```suggestion
```
Personal opinion: while I think this is great advice, I don't think this is the place to put it.

Reply:
+1

Reply:
Maybe on the introduction section?